lib/galaxy/webapps/galaxy/controllers/workflow.py

```diff
@@ -603,6 +603,7 @@ def save_workflow_as(self, trans, workflow_name, workflow_data, workflow_annotat
                     trans,
                     stored_workflow,
                     workflow_data,
+                    from_tool_form=from_tool_form,
```

Comment:
We should move the default for this attribute further down unless it makes sense to be able to set this flag on the controller level.

Reply:
This shouldn't be a controller endpoint at all, but I don't think we should change this right now.

Reply:
Right it will be an api endpoint in the future. Still, we can define the default for from_tool_form further downstream in the call chain.

Reply:
It is, it's just that this function is being passed an unused keyword. This bug can also be fixed by adding `**kwargs` and dropping `from_tool_form` or by changing the caller signature from https://github.com/jmchilton/galaxy/blob/c48be70805990de2d234cb618acdf032a537acda/client/galaxy/scripts/mvc/workflow/workflow-view.js#L692

Reply:
This fix looks good to me for what it is worth. Arguably the web controller should define the default as ``True`` - but since the API controller shouldn't IMO - I think setting it in the client and passing it through this way is the correct thing to do. (Until we actually put the form rendering logic in the client and then all of this could go away, but if it is on the backend we should be specifying we're building special tool states for the form stuff like this.)